TYPO3.Neos/Resources/Public/JavaScript/Content/Inspector/Views/NodeInfoView.js

```diff
@@ -9,7 +9,19 @@ function(
 	template
 ) {
 	Ember.Handlebars.registerBoundHelper('formatDate', function(value) {
-		return new Date(value).toISOString().slice(0, 16).replace('T', ' ');
+		function pad(n) {
+			return n < 10 ? '0' + n : n;
+		}
```

Comment:
Wanted to add it, but as you know it was removed a few weeks ago. Still searching for a good alternative to break our builds with a similar ridiculous dependency.